http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -235,10 +235,10 @@ protected void resolveUrls() {
 
                     OidcProviderMetadata config = getOidcProviderMetadata(discoveryUrl);
 
-                    authUrl = config.getAuthorizationEndpoint();
+                    authUrl = OidcClientUriBuilder.fromUri(config.getAuthorizationEndpoint());
```

Comment:
With this change, `OidcClientUriBuilder` will be used for `authUrl` and `logoutUrl` for the non-Keycloak provider case as well. Is that intended? Will this have any unintended consequences?

Reply:
@fjuma I see, I've fixed this. Thanks!